load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -730,7 +730,6 @@ func (plan *actorLoadTestIncrementPlan) shouldIncrementActorCount(
 	}
 
 	initialSessionNumber := testsession.GetSessionNumberWithDefaultParams(startBlockHeight)
-	// TODO_BLOCKER(@bryanchriswhite): replace with gov param query when available.
```

Comment:
These are non-actionable because `sharedParams` is in scope here which means that we already queried for the on-chain params.